sunpy/map/mapbase.py

```diff
@@ -243,7 +246,51 @@ def __repr__(self):
                                refpix=u.Quantity(self.reference_pixel),
                                refcoord=u.Quantity((self.reference_coordinate.data.lon,
                                                     self.reference_coordinate.data.lat)),
-                               tmf=TIME_FORMAT) + self.data.__repr__()
+                               tmf=TIME_FORMAT)
+
+    def __repr__(self):
+        return object.__repr__(self) + "\n" + self._text_summary() + "\n" + self.data.__repr__()
+
+    def _repr_html_(self):
+        buf = BytesIO()
+        fig = Figure(figsize=(4.8, 4.8), constrained_layout=True)
```

Comment:
This and the below line can be condensed into
```
fig, ax = plt.subplots(figsize=(4.8, 4.8), constrained_layout=True)
```
(and similarly below)

Reply:
Perhaps there's a workaround, but it seems that if I create the figure through `pyplot`, the Jupyter notebook will automatically display the created figure, which is not what I want to happen here.

Reply:
Ah yes, that makes sense.